### ISSUE TITLE
Map Local window showing "Invalid Regex" error

### ISSUE THREAD

**Issue by kartikarora** [ProxymanApp/Proxyman#879]
### Proxyman version? (Ex. Proxyman 1.4.3)
2.24.0
### macOS Version? (Ex. mac 10.14)
11.3.1
### Steps to reproduce 
1. Right click a url
2. Select Tools > Map Local
### Expected behavior
Map local window with response editor should be shown

### Debug Log
```swift
Error Domain=NSCocoaErrorDomain Code=4 "The folder “<url>” doesn’t exist." UserInfo={NSFilePath=/Users/<>username>/Documents/Map Remote Files/<url>.json, NSUserStringVariant=Folder, NSUnderlyingError=0x7f955c362c10 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
```

### Screenshots (optional)
![Screen Shot 2021-05-06 at 2 24 24 pm](https://user-images.githubusercontent.com/6591617/117241559-ccc88780-ae76-11eb-82dc-917254f46bd2.png)

**Comment by NghiaTranUIT** [comment]
Thanks for the bug report. I'm able reproduce it 👍 

To fix it, please go to Preference -> Advance -> Map Local Folder and select a valid folder (Your Desktop).

<img width="702" alt="Screen_Shot_2021-05-06_at_11_26_37" src="https://user-images.githubusercontent.com/5878421/117241738-fb863400-ae5d-11eb-9076-1cb13d33d5a8.png">

Just wondering @kartikarora: Do you folder at `~/Documents/Map Remote Files` exist?

**Comment by kartikarora** [comment]
Quick update, I just realised I had deleted the directory where I store map local files.

Follow up, error message could be refined to show that the directory is missing, and maybe ask user to create it via Proxy Man, or open the "Select Directory" window.

**Comment by NghiaTranUIT** [comment]
Yup, I'm going to fix it. If the folder is invalid, it will fallback to the Desktop Folder 👍

**Comment by kartikarora** [comment]
Too wesy! Thanks for the quick response!

**Comment by NghiaTranUIT** [comment]
Beta build: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_2.24.0_MapLocal_Directory_Not_Found.dmg

We will ship it in the next release 👍 Thank you so much for the bug reporting @kartikarora

**Comment by kartikarora** [comment]
Happy to help!
